TESTS/scripts/Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
```

Comment:
Do we need to commit this file?

Reply:
I think we better do some dependency control over the code generating the tests.
We will have better idea if anything wrong in the future or what's the targeting TF/PyTorch version.